Automattic/sensei
# Add quiz appender filter

### Changes proposed in this Pull Request

* Add filter to the Quiz Appender controls. This is used for the Sensei Pro co-teachers feature.

### Testing instructions

See https://github.com/Automattic/sensei-pro/pull/1934 for testing instructions.


### New/Updated Hooks

* `sensei-lms.Quiz.appender-controls` in `assets/blocks/quiz/quiz-block/quiz-appender.js`
